exercises/practice/atbash-cipher/atbash_cipher_test.odin

```diff
@@ -0,0 +1,171 @@
+package atbash_cipher
+
+import "core:testing"
+
+@(test)
+test_encode_yes :: proc(t: ^testing.T) {
+
+
+	input := "yes"
```

Comment:
Double blank lines in each test case.

Is adding a blank line at the start of a function body your own style? I've been curious about that.